src/transformers/tools/prompts.py

```diff
@@ -14,173 +14,33 @@
-The AI assistant should first explain the tools it will use to perform the task and for what reason, then write the code in Python.
-Each instruction in Python should be a simple assignment. The AI assistant can print intermediate results if it makes sense to do so.
-
-Tools:
-<<all_tools>>
-
-=====
-
-Human: Answer the question in the variable `question` about the image stored in the variable `image`.
-
-Assistant: I will use the tool `image_qa` to answer the question on the input image.
-
-```py
-answer = image_qa(text=question, image=image)
-print(f"The answer is {answer}")
-```
-
-Human: I tried this code, it worked but didn't give me a good result. The question is in French
-
-Assistant: In this case, the question needs to be translated first. I will use the tool `translator` to do this.
-
-```py
-translated_question = translator(question=question, src_lang="French", tgt_lang="English")
-print(f"The translated question is {translated_question}.")
-answer = image_qa(text=translated_question, image=image)
-print(f"The answer is {answer}")
-```
-
-=====
-
-Human: Identify the oldest person in the `document`.
-
-Assistant: I will use the tool `document_qa` to find the oldest person in the document.
-
-```py
-answer = document_qa(document, question="What is the oldest person?")
-print(f"The answer is {answer}.")
-```
-
-Human: Can you generate an image with the result?
-
-Assistant: I will use the tool `image_generator` to do that.
-
-```py
-image = image_generator(answer)
-```
-
-=====
-
-Human: Summarize the text given in the variable `text` and read it out loud.
-
-Assistant: I will use the tool `summarizer` to create a summary of the input text, then the tool `text_reader` to read it out loud.
-
-```py
-summarized_text = summarizer(text)
-print(f"Summary: {summarized_text}")
-audio_summary = text_reader(text=summary)
-```
-
-Human: I got the following error: "The variable `summary` is not defined."
-
-Assistant: My bad! Let's try this code instead.
-
-```py
-summarized_text = summarizer(text)
-print(f"Summary: {summarized_text}")
-audio_summary = text_reader(text=summarized_text)
-```
+CHAT_MESSAGE_PROMPT = """
+Human: <<task>>
 
-Human: It worked! Can you translate the summary in German?
+Assistant: """
 
-Assistant: I will use the tool `translator` to translate the text in German.
 
-```py
-translated_summary = translator(summarized_text, src_lang="English", tgt_lang="German")
-```
+DEFAULT_PROMPTS_REPO = "huggingface-tools/default-prompts"
+PROMPT_FILES = {"chat": "chat_prompt_template.txt", "run": "run_prompt_template.txt"}
 
-====
-"""
 
+def download_prompt(prompt, agent_name, mode="run"):
+    """
+    Downloads and caches the prompt from a repo and returns it contents (if necessary)
+    """
+    if prompt is None:
+        prompt = "huggingface-tools/default-prompts"
```

Comment:
So that it updates here if the global updates

```suggestion
        prompt = DEFAULT_PROMPTS_REPO
```